runtimes/kotlin-1.6/src/main/kotlin/io/openruntimes/kotlin/RuntimeContext.kt

```diff
@@ -0,0 +1,42 @@
+package io.openruntimes.kotlin
+
+import kotlin.Any
+import kotlin.collections.ArrayList
+import kotlin.collections.arrayListOf
+
+import com.google.gson.GsonBuilder
+import com.google.gson.Gson
+
+class RuntimeContext(req: RuntimeRequest, res: RuntimeResponse) {
+    var req: RuntimeRequest
+    var res: RuntimeResponse
+
+    var logs: ArrayList<String>
+    var errors: ArrayList<String>
```

Comment:
Kotlin's idiomatic collection types are `List` and `MutableList`. Since they can be immutable in this context, you can instantiate them here using the helper methods:

```suggestion
    val logs = listOf<String>()
    val errors = listOf<String>()
```

Reply:
Applied, but used `mutableListOf` as it was the only way I found that would pass build with `.add()`.